backend/ops_api/ops/migrations/0002_alter_canfiscalyear_table_and_more.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 4.1.1 on 2022-09-12 15:07
```

Comment:
Do we want a 0002 migration, or just flatten it with 0001; since we're still at our initial stages still?